lib/web/mage/adminhtml/wysiwyg/tiny_mce/tinymce4Adapter.js

```diff
@@ -607,7 +607,7 @@ define([
             }
 
             this.addContentEditableAttributeBackToNonEditableNodes();
-            this.fixRangeSelection(editor);
+            //this.fixRangeSelection(editor);
```

Comment:
I second @krzksz's opinion to remove the line instead of leaving it commented

Reply:
```suggestion
```